### PR TITLE
Fix minor bugs

### PR DESCRIPTION
**Pull request by keelywan**
This fixes the bugs in SearchServlet, NoteGridCard, and SearchBar so that the MVP demo will work.

### CODE REVIEW

**Review by Aradhya-B — APPROVED** [review]
Good job squashin' those bugs!

File: src/main/java/com/google/starfish/servlets/SearchServlet.java [hunk 1]
```diff
@@ -65,13 +65,13 @@ public void doGet(HttpServletRequest req, HttpServletResponse res) throws Servle
   private String getSearchQuery(String school, String course) {
     String stmt = 
         "SELECT * "
-      + "FROM `starfish.notes` "
+      + "FROM `notes` "
       + "WHERE 1=1";
```
Comment (andres-holguin) [inline src/main/java/com/google/starfish/servlets/SearchServlet.java:69]: Lol XD

File: src/main/webapp/components/SearchBar.vue [hunk 2]
```diff
@@ -39,8 +39,7 @@ module.exports = {
       }
       else /* Form validated */ {
         this.error = "";
-        this.queryDatabase(this.school, this.course)
-          .catch(error => this.error = error);
+        this.queryDatabase(this.school, this.course);
```
Comment (andres-holguin) [inline src/main/webapp/components/SearchBar.vue:42]: Doesn't this conflict with commit 0708590 from PR #49?

**Review by andres-holguin — APPROVED** [review]
Bugs begone!

File: src/main/webapp/components/SearchBar.vue (hunk 2 above)
Comment (keelywan) [reply src/main/webapp/components/SearchBar.vue:42]: I think the PR has already been merged

Comment (andres-holguin) [reply src/main/webapp/components/SearchBar.vue:42]: Oh okay, I wasn't sure if this would override that.

**Review by nickKelman — APPROVED** [review]

**Pull request merged**